src/plugins/data_table/dataTables.colReorder.js

```diff
@@ -519,8 +527,10 @@
     ColReorder.aoInstances.push(this);
 
 
+    var tableElement = $(this.s.dt.nTable);
+
     if (this.s.bResizeTableWrapper) {
```

Comment:
this boolean is called resizeTableWrapper but now we're resizing the element itself .. is that desired? Should we rename this option?